tools/importer-rest-api-specs/components/dataapigeneratorjson/models/operations.go

```diff
@@ -0,0 +1,73 @@
+package models
+
+type Operation struct {
+	// Name specifies the Display Name for this Operation (e.g. `Create`) which is also
+	// valid as an identifier.
+	Name string `json:"Name"`
+
+	// ContentType specifies the format of the information being sent with the Operation (e.g. `application/json; charset=utf-8`)
+	ContentType string `json:"ContentType,omitempty"`
+
+	// ExpectedStatusCodes specifies is a list of Status Codes which are expected to be returned (e.g. 200, 201)
+	ExpectedStatusCodes []int `json:"ExpectedStatusCodes,omitempty"`
+
+	// FieldContainingPaginationDetails is a reference to the field within the Response
+	// which contains the pagination details, (e.g. `nextLink`)
+	FieldContainingPaginationDetails string `json:"FieldContainingPaginationDetails,omitempty"`
+
+	// LongRunning specifies if this is a Long Running Operation, meaning that Clients
+	// should follow any `Location` headers to track the result of this operation
+	LongRunning bool `json:"LongRunning,omitempty"`
+
+	// HTTPMethod is the Method used for this operation, (e.g. `GET`, `POST`)
+	HTTPMethod string `json:"HTTPMethod,omitempty"`
+
+	// todo do we need this OptionsObject as well as Options?
```

Comment:
Looks to me like we don't need it? `OptionsObject.Name` seems to be the same as the operation name so perhaps it's redundant?

Reply:
Agreed, the `Options` field should be sufficient here